manifests/params.pp

```diff
@@ -38,10 +38,37 @@
     }
 
     'Debian': {
-      $package_crmsh  = true
-      $package_pcs    = false
-      $package_fence_agents = false
-      $package_install_options = undef
+      case $facts['os']['name'] {
+        'Debian': {
+          if Numeric($facts['os']['release']['major']) > 9 {
+            $package_crmsh = false
+            $package_pcs = true
+            $package_fence_agents = false
+            $package_install_options = undef
+          } else {
+            $package_crmsh  = true
+            $package_pcs    = false
+            $package_fence_agents = false
+            $package_install_options = undef
+          }
+        }
+        'Ubuntu': {
+          if Numeric($facts['os']['release']['major']) > 14 {
+            $package_crmsh = false
+            $package_pcs = true
+            $package_fence_agents = false
+            $package_install_options = undef
+          } else {
```

Comment:
since we only support 16 and newer, we don't need the else block?

Reply:
Well, thus far it still has legacy support for 14.04, thought it would be easy just to keep this in.